examples/aspnet/LandingPage.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace LandingPage.Controllers;
+
+[ApiController]
+[Route("/")]
+public class LandingPageController : ControllerBase
```

Comment:
Dit zal je herkenbaar voorkomen met Quarkus/Express. Dezelfde structuur dus. Met de `[ApiController]` vindt ASP.net dus dynamisch de routes en kunnen we zo de metadata definieren